auth-api/src/auth_api/models/org_status.py

```diff
@@ -24,6 +27,11 @@ class OrgStatus(BaseCodeModel):  # pylint: disable=too-few-public-methods # Temp
 
     __tablename__ = 'org_status'
 
+    @declared_attr
+    def code(cls):  # pylint:disable=no-self-argument, # noqa: N805
```

Comment:
hope this change is also required

Reply:
This is again same mistake someone did earlier. There were lot of places where the model is not upto what's in DB.